app/controllers/reservations_controller.rb

```diff
@@ -295,8 +302,7 @@ def ability_resource
 
   # TODO you shouldn't be able to edit reservations that have passed or are outside of the cancellation period (check to make sure order has been placed)
   def invalid_for_update?
-    params[:id].to_i != @reservation.id ||
-    @reservation.actual_start_at ||
+    different_id = params[:id].to_i != @reservation.id
```

Comment:
Where does `different_id` get used? Should this have a `||` at the end of the line?